pkg/dockerfile/generator.go

```diff
@@ -261,12 +261,17 @@ func (g *Generator) installTini() string {
 	//
 	// N.B. If you remove/change this, consider removing/changing the `has_init`
 	// image label applied in image/build.go.
-	lines := []string{
-		`RUN --mount=type=cache,target=/var/cache/apt set -eux; \
+
+	install := `RUN set -eux; \`
+	if g.Config.Build.GPU {
```

Comment:
It'd be nice to provide some context for why a GPU image needs to run this code. A code comment would help. Or what do you think about updating the `installTini` function to take an `installCurl` Boolean parameter, to make the intention clearer?

Reply:
is this comment enough?

Reply:
That works, yeah. Thanks!